plugin/forward/setup.go

```diff
@@ -211,6 +212,18 @@ func parseBlock(c *caddy.Controller, f *Forward) error {
 		default:
 			return c.Errf("unknown policy '%s'", x)
 		}
+	case "max_queries":
```

Comment:
is there a better name with maybe 'concurrent' in it? (not sure there is, but max_queries is not what's implemented here)

Reply:
changed to `max_concurrent`